### ISSUE TITLE
`prefer-global-this`: support for EyeDropper

### ISSUE THREAD

**Issue by alaychak-hc** [sindresorhus/eslint-plugin-unicorn#2578]
### Description

While it's currently limited to Chrome and Edge, this is (from my knowledge) only supported in browser contexts. Due to this, shouldn't support be added to allow window.EyeDropper (versus automatically changing it to globalThis.EyeDropper)?

If it's due to the fact that it's experimental, feel free to close this. 

[EyeDropper - MDN](https://developer.mozilla.org/en-US/docs/Web/API/EyeDropper)

### Examples

```js
// ❌
if (!globalThis.EyeDropper) {
  return;
}

// ✅
if (!window.EyeDropper) {
  return;
}
```

**Comment by sindresorhus** [comment]
`globalThis` has nothing to do with it being browser or not. Anything not directly related to actual window operations should be on `globalThis`.
